documentation/book/con-customizing-image-pull-policy.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-image-pull-policy-{context}']
+= Customizing Image Pull Policy
+
+{ProductName} allows you to customize the Image Pull Policy for containers in all pods deployed by the Cluster Operator.
```

Comment:
Why is Image Pull Policy in caps?

Reply:
@ncbaratta Well, that is the usual question ... Image Pull Policy is a term from Kubernetes. Which always confuses me with how should I write it. I can change it all to lower case if you want.

Reply:
I think we should lowercase it - unless it's the variable name.

Reply:
Ok, lets change it to lower case then. Thanks.